### PR TITLE
Update expected CSS and rstheme files for ace_invisible

### PR DESCRIPTION
**Pull request by jmcphers**
In https://github.com/rstudio/rstudio/commit/51d1f6d910338b26dda3f7b394f28dc006e7b0db#diff-dd81b5336ba8950d3ec03bb04b55cc27, we added `ace_invisible`, but we didn't update the theme converter tests, so they're all currently broken. 

This change does a few things:

- Regenerates all of the Ace CSS output files to pick up the new rule. There were a lot more changes here than I expected; it looks like the existing output files were a few generations old and were missing semicolons and other CSS formatting. 
- Regenerates all of the rsTheme output files to pick up the new rule. Ditto with the deltas.
- Adds a new script in the tests directory which can be run to update the "expected" results. The idea is that if we make a change like this in the future (in which we *meant* to change the way the generator works), it'll be straightforward to rebuild all the expected results again.

Fixes https://github.com/rstudio/rstudio/issues/4009.

### CODE REVIEW

File: src/cpp/tests/testthat/themes/acecss/active4d.css [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
 
 .ace-active4d .ace_marker-layer .ace_bracket {
   margin: -1px 0 0 -1px;
-  border: 1px solid #BFBFBF
+  border: 1px solid #BFBFBF;
 }
 
 .ace-active4d .ace_marker-layer .ace_active-line {
-  background: rgba(0, 0, 0, 0.071)
+  background: rgba(0, 0, 0, 0.07);
```
Comment (kevinushey) [inline src/cpp/tests/testthat/themes/acecss/active4d.css]: Do we know why these values have changed?

File: src/cpp/tests/testthat/themes/acecss/DebugLine.css [hunk 2]
```diff
@@ -46,60 +46,77 @@
...
 }
 
 .ace-debug-line .ace_variable.ace_parameter {
   font-style: italic;
+
 }
 
 .ace-debug-line .ace_entity.ace_name.ace_tag {
   text-decoration: underline;
+
```
Comment (kevinushey) [inline src/cpp/tests/testthat/themes/acecss/DebugLine.css]: Would be nice to omit the blank lines if possible (fine to leave it if non-obvious)

File: src/cpp/tests/testthat/themes/acecss/blackboard.css [hunk 3]
```diff
@@ -1,46 +1,48 @@
...
 
 .ace-blackboard .ace_marker-layer .ace_bracket {
   margin: -1px 0 0 -1px;
-  border: 1px solid rgba(255, 255, 255, 0.25)
+  border: 1px solid rgba(255, 255, 255, 0.25);
 }
 
 .ace-blackboard .ace_marker-layer .ace_active-line {
-  background: rgba(255, 255, 255, 0.059)
+  background: rgba(255, 255, 255, 0.06);
```
Comment (kevinushey) [inline src/cpp/tests/testthat/themes/acecss/blackboard.css]: Another minor change?

**Comment by kevinushey** [comment]
LGTM but I'm curious why some of the `rgba()` opacities have changed (mostly just by a value of 0.01)

**Comment by jmcphers** [comment]
The reason for this (and other) changes is that several changes to the theme converter have been made since the expected values were updated. For example, a lot of rules that were the result of mixed colors are now `#RRGGBB` instead of `rgba()` due to this change: https://github.com/rstudio/rstudio/commit/bd06774145bec5876224858d3d98c7dfed30b626#diff-183d9a8af37898f52795cc33d097f0c6R255

**Pull request merged**